code/game/objects/structures/windoor_assembly.dm

```diff
@@ -321,27 +322,18 @@
 	//Update to reflect changes(if applicable)
 	update_appearance()
 
-
-
-/obj/structure/windoor_assembly/ComponentInitialize()
-	. = ..()
-	var/static/rotation_flags = ROTATION_ALTCLICK | ROTATION_CLOCKWISE | ROTATION_COUNTERCLOCKWISE | ROTATION_VERBS
-	AddComponent(/datum/component/simple_rotation, rotation_flags, can_be_rotated=CALLBACK(src, .proc/can_be_rotated), after_rotation=CALLBACK(src,.proc/after_rotation))
-
 /obj/structure/windoor_assembly/proc/can_be_rotated(mob/user,rotation_type)
 	if(anchored)
-		to_chat(user, span_warning("[src] cannot be rotated while it is fastened to the floor!"))
+		balloon_alert(user, "need to unwrench")
 		return FALSE
+
 	var/target_dir = turn(dir, rotation_type == ROTATION_CLOCKWISE ? -90 : 90)
```

Comment:
Windoors and windows didn't have flip before and the rotation checks here do not account for it.

Reply:
Good catch!  It also appears that they have their own flip verb to make the windoor inverted.  I will tackle this right now.

Reply:
Fixed with https://github.com/tgstation/tgstation/pull/64441/commits/b141fbde9354c44c7c710126b31213b49ffdbdbb